python/cudf/cudf/_lib/copying.pyx

```diff
@@ -265,3 +270,52 @@ def copy_range(out_col, in_col, int out_begin, int out_end,
     free_column(c_out_col)
 
     return out_col
+
+
+def scatter_to_frames(source, maps):
```

Comment:
Can we add a test for `scatter_to_frames`? (appologies if there is one that I'm missing)

Reply:
This API is definitely not meant to be directly consumed, where we typically build a more user facing API at the DataFrame / Series level and then have tests written against those APIs.

Reply:
@kkraus14 - Perfect.  Then I am personally motivated to submit a followup PR after this gets merged :)